### PR TITLE
Remove SourceBuild.sln, and use ExcludeFromSourceBuild propert…

### PR DESCRIPTION
**Pull request by 333fred**
@tmat @jaredpar for review. @dseefeld @crummel, what changes will this need on the source-build side to consume Roslyn.sln instead of SourceBuild.sln?

### CODE REVIEW

**Comment by dseefeld** [comment]
> what changes will this need on the source-build side to consume Roslyn.sln instead of SourceBuild.sln?
When we get this change in source-build, we'll change our roslyn project to build the new solution.

File: src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.Setup.csproj [hunk 1]
```diff
@@ -56,7 +56,8 @@
           AfterTargets="Build"
           BeforeTargets="SwixBuild"
           DependsOnTargets="_SetSwrFilePath;InitializeDesktopCompilerArtifacts;_PrepareDesktopCompilerArtifactsForOptimization;ApplyOptimizations"
-          Outputs="$(_SwrFilePath)">
+          Outputs="$(_SwrFilePath)"
+          Condition="'$(DotNetBuildFromSource)' != 'true'">
```
Comment (jasonmalinowski) [inline src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.Setup.csproj]: I'm surprised this is running in source build at all, it feels like swr packaging should have been skipped at a much earlier point?

Comment (tmat) [reply src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.Setup.csproj]: It's running because of `AfterTargets="Build"`.

**Comment by 333fred** [comment]
@jaredpar @tmat please review.

**Comment by tmat** [comment]
>     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>

Is this project actually used? It's a legacy project kind.

---
Refers to: src/Compilers/RealParserTests/RealParserTests.csproj:5 in a2a0576. [](commit_id = a2a0576673212b5659ee46f4fb5b44e29dd0408f, deletion_comment = False)

**Comment by 333fred** [comment]
@crummel @dseefeld this is merged, so the next time you pick up source from roslyn master you'll need to change to Roslyn.sln.

**Pull request merged**